From b68e658cdd435af865352a1a23f98700c2fc8956 Mon Sep 17 00:00:00 2001
From: "Chen, Haochuan" <haochuan.z.chen@intel.com>
Date: Tue, 24 Oct 2023 14:52:19 +0800
Subject: [PATCH] Add i2s_boards in avs driver for RPL

Tests Done:
- RPL system with RT5640 audio board bootup and playback works

Tracked-On: OAM-112961
Signed-off-by: Chen, Haochuan <haochuan.z.chen@intel.com>
---
 sound/soc/intel/avs/board_selection.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/sound/soc/intel/avs/board_selection.c b/sound/soc/intel/avs/board_selection.c
index 0dfebf4f6b7cf..d7b6d5248054d 100644
--- a/sound/soc/intel/avs/board_selection.c
+++ b/sound/soc/intel/avs/board_selection.c
@@ -365,6 +365,11 @@ static const struct avs_acpi_boards i2s_boards[] = {
 	AVS_MACH_ENTRY(0xa0c8, avs_tgl_i2s_machines), /* TGL */
 	AVS_MACH_ENTRY(0x4b55, avs_tgl_i2s_machines), /* EHL */
 	AVS_MACH_ENTRY(0x51c8, avs_tgl_i2s_machines), /* ADL-P */
+	AVS_MACH_ENTRY(0x7a50, avs_tgl_i2s_machines), /* RPL-S */
+	AVS_MACH_ENTRY(0x51ca, avs_tgl_i2s_machines), /* RPL-P */
+	AVS_MACH_ENTRY(0x51cb, avs_tgl_i2s_machines), /* RPL-P */
+	AVS_MACH_ENTRY(0x51ce, avs_tgl_i2s_machines), /* RPL-M */
+	AVS_MACH_ENTRY(0x51cf, avs_tgl_i2s_machines), /* RPL-PX */
 	{},
 };
 
-- 
2.40.0

